src/lib/format/ProtoMsgTsdFormatter.ts

```diff
@@ -45,7 +47,7 @@ export namespace ProtoMsgTsdFormatter {
         });
 
         descriptor.getMessageTypeList().forEach((enumType) => {
-            messages.push(MessageFormatter.format(fileName, exportMap, enumType, "", descriptor));
+            messages.push(MessageFormatter.format(fileName, exportMap, enumType, "", descriptor, WellKnownExtensionsMap[fileName]?.extensions));
```

Comment:
Same here, file name is user provided like `examples/book.proto`. It won't be in the `WellKnownExtensionsMap`.

Reply:
See comment [here](https://github.com/agreatfool/grpc_tools_node_protoc_ts/pull/93#discussion_r589059509)